index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dany's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+                <a href="#"class="Logo">My Logo</a>
+                <nav class="Navbar">
+                </nav>
+                <div class="menu"><a href="#"><img src="img/humburger-btn.svg" alt="menu"></a></div>
+    </header>
+    <section>
+            <h1 class="perso">Hey!, my name is Dany<br> I am glad to meet you!</h1>
+            <p class="pro"> I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+
+            <h2 class="connect">LET'S CONNECT</h2>
+            <ul class="icon">
+                <li><a href="www.twitter.com/amiralda"><img src="images/twitter.svg" alt="twitter-icon"></a></li>
+                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="images/linkedin.svg" alt="linkedin-icon"></a></li>
+                <li><a href="https://github.com/amiralda"><img src="images/github.svg" alt="github-icon"></a></li>
+    </section>
+        </div>
```

Comment:
- [ ] KIndly make sure to remove the `div` closing tag because it doesn't have any opening `div` tag.